scripts/apothecary/formulas/FreeImage/FreeImage.sh

```diff
@@ -10,20 +10,20 @@
 FORMULA_TYPES=( "osx" "vs" "win_cb" "ios" "android" )
 
 # define the version
-VER=3160 # 3.16.0
+VER=3170 # 3.16.0
 
 # tools for git use
 GIT_URL=https://github.com/danoli3/FreeImage
-GIT_TAG=3.16.0
+GIT_TAG=3.17.0
```

Comment:
I guess this means we'll make the switch to 3.17.0 @bakercp @arturoc @ofTheo 

I can do the changes for OS X / iOS formula / repo. if this is needed for VS.